src/Calibration.cpp

```diff
@@ -307,22 +307,36 @@ void Calibration::saveCamerasParams() {
  * @brief Save all the 3D object
  *
  * Export 3D points constituting the objects.
- *
+ * Format: X Y Z board_id pts_id
  */
 void Calibration::save3DObj() {
   std::string save_path_object = save_path_ + "calibrated_objects_data.yml";
   cv::FileStorage fs(save_path_object, cv::FileStorage::WRITE);
 
   for (const auto &it_obj : object_3d_) {
+    unsigned int a = 0;
```

Comment:
a more descriptive variable name would be better

Reply:
called it: "obj_pts_count"